src/pydisagg/DisaggModel.py

```diff
@@ -21,574 +21,677 @@ class DisaggModel:
     We do this because it works better with the delta method calculations, and we assume that we
     have a log in the transormations so additive factors become multiplicative
     '''
-
     def __init__(
         self,
-        parameter_transformation: ParameterTransformation,
-        rate_pattern: Optional[NDArray] = None,
-        beta_parameter: Optional[float] = None,
-        error_inflation: Optional[float] = None,
-        beta_standard_error: Optional[float] = None
-    ):
-        """_summary_
+        parameter_transformation: ParameterTransformation
+        ):
+        """Initializes a dissaggregation model
 
         Parameters
         ----------
         parameter_transformation : ParameterTransformation
             Transformation to apply to rate pattern values
-        rate_pattern : Optional[NDArray], optional
-            Rate Pattern to use, should be an estimate of the rates in each bucket
-            that we want to rescale, by default None
-        beta_parameter : Optional[float], optional
-            Additive parameter in log space (acts as multiplier)., by default None
-        error_inflation : Optional[float], optional
-            Inflation factor generated by Delta method., by default None
-        beta_standard_error : Optional[float], optional
-            Standard error of beta parameter, by default None
         """
 
-        self.beta_parameter = beta_parameter
-        self.beta_standard_error = beta_standard_error
-        self.error_inflation = error_inflation
-
         self.parameter_transformation = parameter_transformation
```

Comment:
Why do you need both self.parameter_transformation and self.T?

Reply:
fixed in [commit here](commit ee0ba9e3a212f0fd7abb2bd0f77607a7e3137500)